.github/workflows/build-adoc.yml

```diff
@@ -0,0 +1,21 @@
+name: Build Adoc
+
+on:
+  pull_request:
+    paths:
+      - '*.adoc'
+
+jobs:
+  pre-preview:
+    name: ADOC - Workflow
+    runs-on: ${{ matrix.os }}
+    strategy:
+     matrix:
+        os: [ubuntu-latest, windows-latest]
+        java: [
+          { 'version': '11' },
+          { 'version': '17' }
+        ]
+    steps:
+      - name: Pre preview
+        run: echo Pre preview
```

Comment:
This doesn't do anything. You need to build the docs module and generate the website content.

Reply:
Hi @gastaldi, I changed the file and added the jekyll, I added also gemfile and _config.yml. Let me know if was a mistake and any missing steps.